docs/getting-started.html

```diff
@@ -91,7 +91,7 @@ <h4 class="section-h4">Install</h4>
 			</p>
 
 			<blockquote>
-				<p>If you want to access gravity compiler globally just add it to your <b>PATH</b>!</p>
+				<p>If you want to access the gravity compiler globally just add it to your <b>PATH</b>!</p>
 			</blockquote>
```

Comment:
Is there a reason Gravity isn't capitalized here? It's referring to the language (a proper noun), not the name of an executable. I see this quite a few time throughout this file, and it always jumps out at me (I know, this wasn't actually changed in this PR).